sources/table/index.html

```diff
@@ -15,20 +15,17 @@
                     params="template: 'table-view-context-actions-template', data: $data.viewModel">
                   </abris-dropdown-wrapper>
                 </div>
-                <div class="abrs-advanced-settings">
-                  <div class="abrs-samplesettings-panel-row" data-bind="visible: viewModel.showTools">
-                    <abris-samplesettings-panel params="model: viewModel"></abris-samplesettings-panel>
-                  </div>
-                  <div class="abris-add-table-column" data-bind="visible: $parent.showConstructor">
-                    <abris-dropdown-menu
-                      params="model: viewModel, options: {selectedItem: 'propertyItemToAdd',  getItems: 'getItems', emptyText: 'Открыть связанную колонку', icon: 'add-column' }">
-                    </abris-dropdown-menu>
-                  </div>
+                <div class="advanced-settings">
+                  <abris-samplesettings-panel params="model: viewModel" data-bind="visible: viewModel.showTools">
+                  </abris-samplesettings-panel>
+                  <abris-dropdown-menu class="button-more" data-bind="visible: $parent.showConstructor"
+                    params="model: viewModel, options: {selectedItem: 'propertyItemToAdd',  getItems: 'getItems', emptyText: 'Открыть связанную колонку', icon: 'add-column' }">
+                  </abris-dropdown-menu>
                 </div>
                 <!-- /ko -->
               </div>
-              <div class="table-filter" data-bind="visible: viewFilterTable">
-                <div class="table-filter-container">
+              <div class="filter" data-bind="visible: viewFilterTable">
+                <div class="filter__container">
```

Comment:
All style inside the table markup should be started with the `abris-table`

I'm not sure we need to do it immediately everywhere. The things like `advanced-settings` can be left as is for a while. We'll fix them a bit later when will adopt table header content components.